### DISCUSSION TITLE
How to change the .fancybox-content css in js?

### DISCUSSION THREAD

**Discussion by cq674350529** [fancyapps/ui#384] [Q&A] [answered]
Hi,

Recently, I'd like to add Fancybox to my JekyII generated site, and it works with the following code.
```html
<head>
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fancyapps/ui@4.0/dist/fancybox.css"/>
</head>
<body>
  <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.1/dist/jquery.min.js"></script>
  <script src="https://cdn.jsdelivr.net/npm/@fancyapps/ui@4.0/dist/fancybox.umd.js"></script>
  <script>
        $(document).ready(function() {
            $("p img").each(function() {
                var strA = "<a data-fancybox='gallery' href='" + this.src + "'></a>";
                $(this).wrapAll(strA);
            });

            Fancybox.bind('[data-fancybox="gallery"]', {
            });
        });
  </script>
</body>
```

The picture looks well in web page.
![image](https://user-images.githubusercontent.com/13868458/206897838-15d1f68d-9935-4bbf-81dd-cbe2c0594346.png)

However it looks weird when clicking the picture, due to its transparent background.
![image](https://user-images.githubusercontent.com/13868458/206897871-7599229e-d6a3-4adb-bc2b-fe046071dbb9.png)

Previously, I came across this same issue when using Hexo + Next theme, and solved it by modifying the ".fancybox-content".
```css
.fancybox-content {
  background: #fff !important;
}
```

So how can I achieve the same purpose with javascript? I looked at the offcial documents, and didn't find a solution.

Thanks in advance!

**Comment by fancyapps** [comment, marked as answer]
Hi,

The answer is very simple - use callbacks, you can access container element (`slide.$el`) and wrapping element (`slide.$content`) from almost any callback. There are many demos on the home page about using callbacks, did you find it difficult to use them or something was not clear in the docs?

**Reply by cq674350529** [reply]
Hi,

Thanks for your suggestions. I'll check the demos again. Previously, I didn't know which attributes or functions are available in callbacks. I'm not familiar with front-end development.

Thanks!

**Reply by cq674350529** [reply]
According to your suggestions, I solved it successfully.

Thanks!
